### ISSUE TITLE
List item .image styles affecting "file image outline" icon

### ISSUE THREAD

**Issue by rijk** [Semantic-Org/Semantic-UI#4459]
(small issue) I found one of my list items was out of alignment. HTML:

```
<div class="ui relaxed list">

                    <div class="item">
                      <i class="large file image outline icon"></i>
                      <div class="content">
                        <div class="header">Packing list</div>
                        <div class="description">nopic_192.gif</div>
                      </div>
                    </div>

                    <div class="item">
                      <i class="large file pdf outline icon"></i>
                      <div class="content">
                        <div class="header">Packing list</div>
                        <div class="description">tizenfordummies.pdf</div>
                      </div>
                    </div>

                </div>
```

The culprit turned out to be this rule in `list.import.less`:

```
.ui.list .list > .item > .image:not(:only-child):not(img),
.ui.list > .item > .image:not(:only-child):not(img) {
  padding-right: @imageDistance;
}
```

**Comment by domialex** [comment]
Same thing happens with a "block header" with an icon of "image something something", there's some mixing up happening between styles.

**Comment by rijk** [comment]
This is not stale! Still an issue.

**Comment by lubber-de** [comment]
I think, this is fixed with the shortly upcoming 2.7.5 of Fomantic-UI by https://github.com/fomantic/Fomantic-UI/pull/714

See your adjusted example: https://jsfiddle.net/kxm2z8ab/
